main/FatesInterfaceMod.F90

```diff
@@ -369,6 +369,9 @@ subroutine allocate_bcin(bc_in, nlevsoil_in, nlevdecomp_in, num_lu_harvest_cats)
       if (hlm_use_lu_harvest .gt. 0) then
          allocate(bc_in%hlm_harvest_rates(num_lu_harvest_cats))
          allocate(bc_in%hlm_harvest_catnames(num_lu_harvest_cats))
+      else ! LoggingMortality_frac needs these passed to it regardless of harvest
+         allocate(bc_in%hlm_harvest_rates(0))
```

Comment:
huh, never thought of doing this, but I suppose it makes sense.   I hope that the various weird compilers out there don't have problems with this.

Reply:
Hmmm, good point.  I haven't actually tracked down why this is an issue for fates_main_api versus fates_next_api, but my assumption is that it has something to do with a change in compiler options that might have taken place during the change from cime5.6 to cime5.8.